onnxruntime/core/providers/cuda/tensor/gather_elements.h

```diff
@@ -23,6 +23,7 @@ struct GatherScatterElementsArgs;
 void CoalesceDimensions(TensorShapeVector& input_shape, TensorShapeVector& indices_shape,
                         TensorShapeVector* p_indices_strides, int64_t axis, GatherScatterElementsArgs& args);
 ONNX_NAMESPACE::TensorProto_DataType GetElementType(size_t element_size);
```

Comment:
This one doesn't look right... How come we decide element type from element size...